src/config.ts

```diff
@@ -2,15 +2,17 @@ import { Team } from './data/types';
 
 const teams: Team[] = [
   {
-    name: 'Team1',
+    name: 'Sync',
     project: {
-      name: 'Placeholder1',
-      description: '',
+      name: 'Sync',
+      description: ':tv: Watch YouTube videos interactively in sync with friends',
       images: {
-        bannerURI: undefined,
+        bannerURI: 'https://github.com/ubclaunchpad/sync/blob/master/.static/banner.png',
```

Comment:
Post-PR note: I had to change this after the fact because we need a direct image link. See https://ubclaunchpad.github.io/new/config/globals.html#project

tldr:

```dif
- https://github.com/ubclaunchpad/inertia/blob/master/.static/inertia-with-name.png
+ https://raw.githubusercontent.com/ubclaunchpad/inertia/master/.static/inertia-with-name.png
```